code/game/objects/items/weapons/chaplain_items.dm

```diff
@@ -259,3 +311,67 @@
 	else
 		icon_state = "assunzionesheath_empty"
 
+/obj/item/storage/altar
+	name = "altar"
+	desc = "A small portable altar."
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "talismanaltar"
+	can_hold = list(/obj/item/nullrod)
+	storage_slots = 1
+	drop_sound = 'sound/items/drop/axe.ogg'
+	pickup_sound = 'sound/items/pickup/axe.ogg'
+
+/obj/item/storage/altar/attack_hand(mob/user)
+	if(!isturf(loc))
+		..()
+	if(Adjacent(user))
+		open(user)
+		return TRUE
+	return FALSE
+
+/obj/item/storage/altar/MouseDrop(mob/user as mob)
+	if((user == usr && (!use_check(user))) && (user.contents.Find(src) || in_range(src, user)))
```

Comment:
User here can't really not be the proc user, I think. Should also use_check_and_message this